test/malli/clj_kondo_test.cljc

```diff
@@ -63,11 +75,27 @@
          {'kikka
           {:arities {1 {:args [:int],
                         :ret :int},
-                     :varargs {:args [:int :int {:op :rest, :spec :int}],
```

Comment:
This is a real varargs case and should generate `:rest` definition instead `:seqable`. From clj-kondo:

```markdown
{:op :rest, :spec :int}. This can be used to match remaining arguments in vararg signatures. 
```

Reply:
I see!  Thanks.